### DISCUSSION TITLE
How to tell if a flag was set or not?

### DISCUSSION THREAD

**Discussion by Integralist** [urfave/cli#1804] [Q&A] [answered]
Hello,

Is it possible to tell if a flag has been set or not? 

For example, a user might want an integer flag value to be zero and so I can't rely on the zero value of an integer to determine if it was sent or not. In Go you typically use a pointer so you can tell if the value was provided vs a zero value.

Is there anything like this available?

Thanks.

**Comment by dearchap** [comment, marked as answer]
In v2 you can use context.IsSet(flagName) to determine if the flag has been set.

**Reply by Integralist** [reply]
Awesome! Thank you 🙂
